_posts/2018-09-01-generate-pulp-3-python-client-using-swagger-codegen.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Generate Pulp 3 Python client using swagger-codegen
+author: Dennis Kliban
+---
```

Comment:
Add an introduction to explain what the bindings are and what they can be used for. Maybe:

> As a third party wishing to integrate with Pulp 3, you can use OpenAPI tooling to generate bindings for over 20 different languages. Here is an example of how to use `swagger-codegen` to generate Python bindings.